src/reanimated2/hook/useReducedMotion.web.ts

```diff
@@ -1,3 +1,4 @@
+// ts-prune-ignore-next
```

Comment:
Could we add a rule so a comment is necessary when adding `ts-prune` directive like this?

Reply:
We would have to create custom eslint rule 😞 so I don't think it's worth the effort. I've added comments to all the existing ts-prune-ignore-next occurences.